src/main/java/seedu/address/logic/orderer/FoodSelector.java

```diff
@@ -89,8 +90,10 @@ private float calculateScore(Food food, Set<Allergy> userAllergies) {
         }
 
         score = 1;
-        score *= scoreFromRating(food.getRating());
-        score *= scoreFromPrice(food.getPrice());
+        score *= scoreFromRating(food.getRating()) + SCORE_BUFFER;
+        score *= scoreFromPrice(food.getPrice()) + SCORE_BUFFER;
+
+        assert(score > 0);
```

Comment:
Is there a point in asserting anymore? To prevent rating / price being negative?

Reply:
just so developers don't make the same mistake in future!

Reply:
oh right, that was part of the diff :) Cool!